test/unit/spec/OktaSignIn_spec.js

```diff
@@ -386,4 +410,23 @@ Expect.describe('OktaSignIn v2 bootstrap', function () {
       });
     });
   });
-});
+
+  Expect.describe('Directly consumes proxyIdxResponse from widget config and do not introspect state token', function () {
+    itp('Gets device enrollment values from proxyIdxResponse and render mdm device enrollment terminal view', function () {
```

Comment:
it's not relevant here to mention anything about `device enrollment`.. just say `gets proxy idx response and render terminal view`